src/transformers/modeling_ctrl.py

```diff
@@ -264,7 +275,7 @@ class CTRLModel(CTRLPreTrainedModel):
     def __init__(self, config):
         super().__init__(config)
         self.output_hidden_states = config.output_hidden_states
-        self.output_attentions = config.output_attentions
+        self.output_attentions = False
```

Comment:
`output_attentions` should not be saved in any model class - we should be able to always switch `output_attentions` on and off in the forward pass